web/src/components/troubleshoot/PreflightsProgress.tsx

```diff
@@ -63,7 +79,7 @@ export default function PreflightProgress(props) {
           </h1>
           <div className="flex alignItems--center u-marginTop--20">
             <span className="u-fontWeight--bold u-fontSize--normal u-textColor--secondary u-marginRight--10">
-              {percentage + "%"}
+              {percentage > 0 ? `${percentage}%` : "0%"}
```

Comment:
before `percentage` was a module scoped var that would be updated in a few spots. Got rid of that. 

https://github.com/replicatedhq/kots/blob/74cc0ca690f70946b47584c6813166caac1663ca/web/src/components/troubleshoot/PreflightsProgress.jsx#L21-L43